ctf/org.eclipse.tracecompass.ctf.core/src/org/eclipse/tracecompass/ctf/core/event/types/IntegerDeclaration.java

```diff
@@ -392,6 +418,26 @@ public boolean isSigned() {
         return fSigned;
     }
 
+    /**
+     * Is the integer a varint ?
+     *
+     * @return a boolean indicating varint status
+     * @since 4.4
```

Comment:
Requires plug-in Bundle-Version to be stepped to 4.5

Reply:
Done 👍